datasets/us/navy/crawler.py

```diff
@@ -0,0 +1,129 @@
+from lxml import html, etree
+import orjson
+from urllib.parse import urljoin
+
+from zavod import Context, helpers as h
+from zavod.logic.pep import categorise
+from zavod.shed.zyte_api import fetch_html
+
+BASE_URL = "https://www.navy.mil"
+API_URL = "https://www.navy.mil/API/ArticleCS/Public/GetList?moduleID=709&dpage=%d&TabId=119&language=en-US"
+
+
+def emit_person(context: Context, country: str, source_url: str, role: str, name: str):
+    person = context.make("Person")
+    person.id = context.make_id(country, name, role)
+    person.add("name", name)
+    person.add("position", role)
+    person.add("sourceUrl", source_url)
+
+    position = h.make_position(context, role, country=country, topics=["mil"])
+
+    categorisation = categorise(context, position, is_pep=True)
+    if categorisation.is_pep:
+        occupancy = h.make_occupancy(context, person, position)
+        if occupancy:
+            context.emit(person, target=True)
+            context.emit(position)
+            context.emit(occupancy)
+
+
+def crawl_person(context: Context, item_html: str) -> None:
+    doc = html.fromstring(item_html)
+    link_el = doc.find(".//a")
+    if link_el is None:
+        return
+    url = link_el.get("href")
+    name = link_el.find(".//h2").text_content().strip()
```

Comment:
can we move typical ranks or whatecer like `admiral` to `title`, and if all names have a title, warn if we don't find one in a name?

```
sort data/datasets/us_navy/names.txt |head
Acting Under Secretary of the Navy
Adm. Lisa Franchetti
Admiral Christopher Grady
Admiral Daryl Caudle
Admiral James W. Kilby
Admiral Lisa Franchetti
Admiral Samuel Paparo
Admiral Stephen T. "Web" Koehler
Admiral Stuart Munsch
Admiral William Houston
```